scripts/templates/milestone-news-item.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "RDF4J ${MVN_VERSION_RELEASE} Milestone ${M}"
+date: ${DATETIME}
+layout: "single"
+categories: ["news"]
+---
+Milestone number ${M} of the upcoming ${MVN_VERSION_RELEASE} release of RDF4J is now available for download.
+
+RDF4J ${MVN_VERSION_RELEASE} is a minor release focusing on ... .
+
+This milestone build is not yet feature-complete, but we are putting it out to receive early feedback on all the improvements we have put in.
+
+<!--more-->
+
+ - [issues fixed in ${MVN_VERSION_RELEASE} Milestone 1](https://github.com/eclipse/rdf4j/issues?q=is%3Aissue+label%3AM${M}+is%3Aclosed+milestone%3A${MVN_VERSION_RELEASE})
```

Comment:
This will only work if the closed issues have been labeled with the "M1" ("M2", etc.) tag.

Reply:
I also noticed I forget to add a $M variable to the label too.